<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A CS123 Final</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>270</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 183, 130);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="cameraLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>BÃ©zier Camera</string>
          </property>
         </widget>
         <widget class="QLabel" name="heightLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>100</width>
            <height>30</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Camera Height</string>
          </property>
         </widget>
         <widget class="QSlider" name="heightSlider">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>40000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="heightInput">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>120</y>
            <width>51</width>
            <height>24</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QSlider" name="scaleSlider">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>5500</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="scaleInput">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>170</y>
            <width>51</width>
            <height>24</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="scaleLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>100</width>
            <height>30</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Path Scale</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="pathOneButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>100</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Path One</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="pathTwoButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>220</y>
            <width>100</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Path Two</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="pathThreeButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>100</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Path Three</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="eggButton">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Egg</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="contentButton">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>70</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>More Egg content</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="motionCheck">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Pause Motion</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>210</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 183, 130);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <widget class="QLabel" name="fStopLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>100</width>
            <height>30</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>f-Stop</string>
          </property>
         </widget>
         <widget class="QSlider" name="fStopSlider">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>40000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="dofLabel">
          <property name="geometry">
           <rect>
            <x>7</x>
            <y>10</y>
            <width>171</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Depth Of Field</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="fStopEdit">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>100</y>
            <width>51</width>
            <height>24</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="dofEnableCheckbox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>161</width>
            <height>22</height>
           </rect>
          </property>
          <property name="text">
           <string> Enable Depth of Field</string>
          </property>
         </widget>
         <widget class="QSlider" name="planeInFocusSlider">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>121</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>5500</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="planeInFocusEdit">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>170</y>
            <width>51</width>
            <height>24</height>
           </rect>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="planeInFocusLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>100</width>
            <height>30</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Plane in Focus</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 183, 130);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="mappingLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Enable Mappping</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="textureCheck">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>87</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Texture</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="displacementCheck">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Displacement</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="View" name="view" native="true">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QWidget</extends>
   <header>view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
